src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
 // write code here
+/* eslint-disable */
+const tHead = document.querySelectorAll('table thead th');
```

Comment:
```suggestion
const tHead = document.querySelectorAll('th');
```
enough just `th` selector